dashboard/app/helpers/ai_diff_bedrock_helper.rb

```diff
@@ -38,6 +38,14 @@ def self.get_prompt_for_context(context, course_name, unit_name, lesson_name)
       Here are the search results in numbered order:
       $search_results$
 
+      $output_format_instructions$", course_name: course_name, unit_name: unit_name, lesson_name: lesson_name
+      )
+    when SharedConstants::AI_DIFF_CONTEXT[:GENERAL]
+      prompt = format("You are a teaching assistant named Aida. It's your job to help K-12 computer science teachers using the code.org platform plan their lessons and adjust lesson plans to fit class time requirements, help students that are ahead or behind, provide alternate explanations of the material, and other relevant teaching tasks. Your responses should be warm and helpful because you're the best lesson planner there could be, and you know all about computer science education.
```

Comment:
I just modified this from `COURSE`.  Again, I realize it might make sense to modify this for a `GENERAL` context.

Reply:
Sounds good! [ I just created this ticket for the task.](https://codedotorg.atlassian.net/jira/software/projects/AITT/boards/70?selectedIssue=AITT-943)